site/content/docs/monitoring/check-results.md

```diff
@@ -30,6 +30,13 @@ Single check runs can be accessed by selecting them on the time ribbon or by hov
 
 Selecting a check execution will take you to the dedicated check result page for it, which will look different based on whether you had been running a [Browser](#browser-check-results) or an [API check](#api-check-results).
 
+When retries are enabled, an additional icon highlights that the check result contains multiple check runs. All attempts as well as the final result are available to view.
+
+The check retries ratio indicates how many of the total checks run were executed twice due to an initial failed attempt.
```

Comment:
I'm not sure I like this suggestion more TBH - it's just a way to get around having to update this again if we add the capability of having more than 1 retry per check run.
```suggestion
The check retries ratio indicates how many of the total checks run were retried due to an initial failed attempt.
```